docs/perf_test_result_dumps.md

```diff
@@ -5,7 +5,7 @@
 Command:
 
 ```
-mix run --no-start -e 'OMG.Performance.start_simple_perftest(8_000, 32, %{block_every_ms: 15_000})'
+DD_DISABLED=true mix run --no-start -e 'OMG.Performance.start_simple_perftest(8_000, 32, %{block_every_ms: 15_000})'
```

Comment:
1. in order for me to work, I needed `APP_ENV=localwatcher` too.
2. it seems we have this command copy-pasted many times here. Maybe let's remove the exact copies and rely on the "Command as above"
3. more general remark on the system environment being required to be set - wouldn't we rather put the values required in invocations like these into the defaults, as held in `config/dev.exs`.

Reply:
3. I'm doing that https://github.com/omisego/elixir-omg/pull/884 in progress.